public/components/datasources/components/__tests__/testing_constants.ts

```diff
@@ -3,6 +3,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { CachedAcceleration } from '../../../../../common/types/data_connections';
```

Comment:
All the. testing constants should be moved to `https://github.com/opensearch-project/dashboards-observability/blob/main/test/datasources.ts`

Reply:
done.